tests/testthat/test_functions.R

```diff
@@ -0,0 +1,12 @@
+context("Tests of package functions")
+
+testthat::test_that("sum_columns returns vector of correct length", {
+  testthat::expect_length((sum_columns(iris)), 4)
```

Comment:
I think we should create mock data for tests. `iris` comes from external package and can be changed in future.